components/prism-factor.js

```diff
@@ -0,0 +1,370 @@
+(function (Prism) {
+
+	var comment_inside = {
+		function: {
+			pattern: /(\b)(?:TODO|FIX|FIXME|NOTE|BUG|XX+|HACK|\?\?+|!!+)(?=\b|:)/,
+			lookbehind: true
+		}
+	};
+	var string_inside = {
+		number: /(?:\\[^\s']|%\w)/
```

Comment:
Unnecessary non-capturing group.

```suggestion
		number: /\\[^\s']|%\w/
```